paasta_tools/paasta_metastatus.py

```diff
@@ -147,13 +176,84 @@ def utilization_table_by_grouping_from_mesos_state(
         table_rows.append(metastatus_lib.get_table_rows_for_resource_info_dict(
             [v for g, v in grouping_values],
             healthcheck_utilization_pairs,
-        ) + [str(resource_info_dict['slave_count'])])
+        ))
+
+        if service_instance_stats:
+            # Calculate the max number of runnable service instances given the current resources (e.g. cpus, mem, disk)
```

Comment:
How about putting this into a function?

Reply:
Good idea. Did so and added a test for it.